embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/session/SessionHandler.kt

```diff
@@ -233,126 +279,44 @@ internal class SessionHandler(
     }
 
     /**
-     * It builds an end active session message, it sanitizes it, it performs all types of memory cleaning,
-     * it updates cache and it sends it to our servers.
-     * It also stops periodic caching and automatic session stopper.
+     * 'Ends' the active session. Note that this logic is also used for caching the session
+     * periodically so the session won't always end. The behavior is controlled by the
+     * [SessionEndType] passed to this function.
      */
-    private fun runEndSessionFull(
-        endType: SessionLifeEventType,
-        originSession: Session?,
+    private fun runEndSessionImpl(
```

Comment:
Since we aren't always really ending the session, what do you think about renaming this to indicate that it's just taking a current snapshot of the session?

Reply:
Renamed this function